feature/src/main/java/com/example/feature/countrylist/componenets/CountrySelectingList.kt

```diff
@@ -26,7 +27,9 @@ fun CountryListList(
     ) {
         list.forEach { continent ->
             stickyHeader {
-                Text(text = continent.name, style = MaterialTheme.typography.h5,)
+                Surface(Modifier.fillParentMaxWidth()) {
```

Comment:
Hello Sarah! Thank you for taking care of our application! Love this!

I've just a minor suggestion, instead of Surface you could just set a background for this Text, such as:

```
Text(modifier = Modifier.fillMaxWidth().background(color = MaterialTheme.colors.surface)
```

The Surface element is totally fine here, but I see we could use it if the header required a more complex UI, since it is just a Text we can just have a colored background. Also usually the Surface component is used for complex structures such as Custom Buttons, Cards and etc.

What do you think?

Reply:
Yes - makes sense to me; I agree! Changes made 👍
Thanks. I'm continuing to review this app - very interesting stuff...I'm creating my own demo based on it.

Reply:
Oh - one more thing...I reformatted which messed up the indents, let me know if you need me to adjust per your style guide...

Reply:
Your changes looks good to me.

Any suggestion/feature/improvement that you may have will be very much appreciated. 

Thank you, Sarah!